pkgs/development/python-modules/xsdata/default.nix

```diff
@@ -48,7 +48,8 @@ buildPythonPackage rec {
     ];
   };
 
-  propagatedBuildInputs = [ setuptools typing-extensions ];
+  buildInputs = [ setuptools ];
```

Comment:
```suggestion
  nativeBuildInputs = [ setuptools ];

```

Reply:
Ouch, you're absolutely right, sorry!

Reply:
native *build* inputs
Also, please squash.

Reply:
Heh, guess I was more tired than I thought I was, sorry. Fixed now and squashed %)